### DISCUSSION TITLE
Five server - connecting msg in console browser

### DISCUSSION THREAD

**Discussion by darianmorat** [yandeu/five-server#120] [Help / Questions]
![Screenshot 2024-01-10 115416](https://github.com/yandeu/five-server/assets/80854633/e1b8448f-93a4-467b-bad1-4b2aed1868c9)

I love the extension, but this msg is so annoying, every time I get confused cause it is always loading after the console.log. Is there any way to remove that?

**Comment by yandeu** [comment]
There is a silent option, but I don't remember if it hides those messages.
